src/App.jsx

```diff
@@ -12,6 +12,7 @@ const App = () => {
         <Route path="/" element={<Home />} />
         <Route path="/registration/:event" element={<Registration />} />
       </Routes>
+      <GalleryCarousel />
```

Comment:
Move GalleryCarousal to the Home page component. As it will be there in Home page just